src/app/tab/tab.component.html

```diff
@@ -1,479 +1,804 @@
-    [style.backgroundImage]="'url('+ shared.bg +')'"
+    [style.backgroundImage]="'url(' + shared.bg + ')'"
     [style.filter]="getBrightness()"
     [style.backgroundSize]="getBgSize()"
     [style.backgroundBlendMode]="getBgBlend()"
-    [style.backgroundRepeat]="settings.config.design.imageSize === 10 ? 'repeat' : 'no-repeat'">
-  </div>
-  
-  <tab-bookmarks></tab-bookmarks>
+    [style.backgroundRepeat]="
+      settings.config.design.imageSize === 10 ? 'repeat' : 'no-repeat'
+    "
+  ></div>
 
-  <div class="tabGrid" [ngClass]="{'tabGridBookmarksBar': settings.config.bookmark.enabled}">
+  <tab-bookmarks></tab-bookmarks>
 
+  <div
+    class="tabGrid"
+    [ngClass]="{ 'tabGridBookmarksBar': settings.config.bookmark.enabled }"
+  >
     <article id="nw" class="gridPosition">
-      <div 
-        class="forceLineBreak" 
-        *ngIf="settings.config.quickLink.position === 'nw' && settings.config.quickLink.enabled" 
-        [ngStyle]="setQuickLinkWrapStyles()">
-        <tab-quickLinks
-          [ngStyle]="setQuickLinkStyles()">
-        </tab-quickLinks>
+      <div
+        class="forceLineBreak"
+        *ngIf="
+          settings.config.quickLink.position === 'nw' &&
+          settings.config.quickLink.enabled
+        "
+        [ngStyle]="setQuickLinkWrapStyles()"
+      >
+        <tab-quickLinks [ngStyle]="setQuickLinkStyles()"> </tab-quickLinks>
       </div>
       <ng-container *ngIf="settings.config.time.clocks.length > 0">
-        <ng-container *ngFor="let clock of settings.config.time.clocks; let i = index;">
-          <tab-time 
-            *ngIf="clock.position === 'nw'" 
-            [index]="i" 
-            [clock]="clock" 
-            [ngStyle]="setTimeStyles(clock)">
+        <ng-container
+          *ngFor="let clock of settings.config.time.clocks; let i = index"
+        >
+          <tab-time
+            *ngIf="clock.position === 'nw'"
+            [index]="i"
+            [clock]="clock"
+            [ngStyle]="setTimeStyles(clock)"
+          >
           </tab-time>
         </ng-container>
       </ng-container>
-      <ng-container *ngIf="settings.config.covidData.enabled && settings.config.covidData.countries.length > 0">
-        <ng-container *ngFor="let c of settings.config.covidData.countries; let i = index;" >
-          <tab-covid *ngIf="c.position === 'nw'" [index]="i" [countryCode]="c.code"
-          [style.order]="shared.getOrder(c.id)"
-          [style.fontSize]="shared.getFontSize(c.textScaling, 20)"
-          [style.transform]="shared.getOffset(c.offset)"></tab-covid>
+      <ng-container
+        *ngIf="
+          settings.config.covidData.enabled &&
+          settings.config.covidData.countries.length > 0
+        "
+      >
+        <ng-container
+          *ngFor="let c of settings.config.covidData.countries; let i = index"
+        >
+          <tab-covid
+            *ngIf="c.position === 'nw'"
+            [index]="i"
+            [countryCode]="c.code"
```

Comment:
You need to swap the style attributes here for the `[ngStyle]="setCovidCountryStyles(c)"`, make sure it is done for every position.

Reply:
Fixed